src/processing/dom/parse5-dom-adapter.js

```diff
@@ -11,16 +11,25 @@ export default class ServerDomAdapter extends BaseDomAdapter {
     }
 
     getAttr (el, attr) {
-        return el.attribs[attr];
+        for (var i = 0; i < el.attrs.length; i++) {
+            if (el.attrs[i].name === attr)
+                return el.attrs[i].value;
+        }
+
+        return void 0;
```

Comment:
We should return `null` here. Since `void 0` is used for something what's "not defined"